File: blog-service/src/main/java/net/blueshell/blogservice/service/BlogService.java
package net.blueshell.blogservice.service;

import net.blueshell.blogservice.model.Blog;
import net.blueshell.blogservice.repository.BlogRepository;
import net.blueshell.db.BaseModelService;
import org.springframework.beans.factory.annotation.Autowired;

import java.util.UUID;

@org.springframework.stereotype.Service
public class BlogService extends BaseModelService<Blog, UUID, BlogRepository> {

    @Autowired
    public BlogService(BlogRepository blogRepository) {
        super(blogRepository);
    }
}

File: api/src/main/kotlin/net/blueshell/api/feature/blog/service/BlogService.kt
package net.blueshell.api.feature.blog.service

import net.blueshell.api.feature.blog.model.Blog
import net.blueshell.api.feature.blog.repository.BlogRepository
import net.blueshell.api.shared.service.BaseModelService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.context.ApplicationEventPublisher
import org.springframework.stereotype.Service

@Service
class BlogService @Autowired constructor(blogRepository: BlogRepository, events: ApplicationEventPublisher) :
    BaseModelService<Blog, Long, BlogRepository>(blogRepository)
